contributing.md

```diff
@@ -0,0 +1,306 @@
+Step 1/7 : FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+ ---> 75a64ccf990b
+Step 2/7 : ENV OPERATOR=/usr/local/bin/capsule     USER_UID=0     USER_NAME=capsule
+ ---> Using cache
+ ---> e4610bd8596f
+Step 3/7 : COPY build/_output/bin/capsule ${OPERATOR}
+ ---> Using cache
+ ---> 1f6196485c28
+Step 4/7 : COPY build/bin /usr/local/bin
+ ---> Using cache
+ ---> b517a62ca352
+Step 5/7 : RUN  /usr/local/bin/user_setup
+ ---> Using cache
+ ---> e879394010d5
+Step 6/7 : ENTRYPOINT ["/usr/local/bin/entrypoint"]
+ ---> Using cache
+ ---> 6e740290e0e4
+Step 7/7 : USER ${USER_UID}
+ ---> Using cache
+ ---> ebb8f640dda1
+Successfully built ebb8f640dda1
+Successfully tagged quay.io/clastix/capsule:latest
+INFO[0004] Operator build complete. 
+```
+
+The image `quay.io/clastix/capsule:latest` will be available locally, you just
+need to push it to kind with the following command.
+
+```
+# kind load docker-image --nodes capsule-control-plane --name capsule quay.io/clastix/capsule:latest
+Image: "quay.io/clastix/capsule:latest" with ID "sha256:ebb8f640dda129a795ddc68bad125cb50af6bfb8803be210b56314ded6355759" not yet present on node "capsule-control-plane", loading...
+```
+
+### Deploy the Kubernetes manifests
+
+With the current `kind-capsule` context enabled, create the `capsule-system`
+Namespace that will contain all the Kubernetes resources.
+
+```
+# kubectl create namespace capsule-system
+namespace/capsule-system created
+```
+
+Now it's time to install the _Custom Resource Definition_:
+
+```
+# kubectl apply -f deploy/crds/capsule.clastix.io_tenants_crd.yaml
+customresourcedefinition.apiextensions.k8s.io/tenants.capsule.clastix.io created
+```
+
+Finally, install the required manifests issuing the following command:
+
+```
+# kubectl apply -f deploy
+mutatingwebhookconfiguration.admissionregistration.k8s.io/capsule created
+clusterrole.rbac.authorization.k8s.io/namespace:deleter created
+clusterrole.rbac.authorization.k8s.io/namespace:provisioner created
+clusterrolebinding.rbac.authorization.k8s.io/namespace:provisioner created
+deployment.apps/capsule created
+clusterrole.rbac.authorization.k8s.io/capsule created
+clusterrolebinding.rbac.authorization.k8s.io/capsule-cluster-admin created
+clusterrolebinding.rbac.authorization.k8s.io/capsule created
+secret/capsule-ca created
+secret/capsule-tls created
+service/capsule created
+serviceaccount/capsule created
+```
+
+You can check if Capsule is running checking the logs:
+
+```
+# kubectl -n capsule-system logs -f -l name=capsule
+...
+{"level":"info","ts":1596125071.5951712,"logger":"controller-runtime.controller","msg":"Starting workers","controller":"tenant-controller","worker count":1}
+```
+
+Since Capsule is built using _OperatorSDK_, logging is handled by the zap
+module: verbosity increase can be controlled using the CLI flag `--zap-level`
+with a value from `0` to `10` or the [basic keywords](https://godoc.org/go.uber.org/zap/zapcore#Level).
+
```

Comment:
Ok. Please note log level 0 is not supported

kaas~/Shared/fastweb/kubernetes/k8s-capsule$ kubectl -n capsule-system logs -f capsule-665899b69-lrwcn
invalid argument "0" for "--zap-level" flag: invalid log level "0"
invalid argument "0" for "--zap-level" flag: invalid log level "0"

Reply:
Thanks, fixed and going to merge!